### DISCUSSION TITLE
Product Review App/Plugin Ideas

### DISCUSSION THREAD

**Discussion by jwhawkes** [saleor/saleor#11483] [Q&A] [answered]
Has anyone got any advice or recommendations for implementation product reviews for products?

**Comment by lkostrowski** [comment, marked as answer]
Hi @jwhawkes 

Recently I've created a repository where we can gather more community around apps:
https://github.com/saleor/apps/discussions

Lets open discussion there and describe your idea. As Saleor we write some official apps, so maybe we can pick that one too, but we'd like to know what you'd like to achieve

**Comment by timuric** [comment]
@jwhawkes if you wish to simply highlight some product reviews from external sources like typeform, you can simply use attributes to create such reviews. If you are looking for a full fledged product review system like Trustpilot, you could fetch reviews in your storefront by matching them with URL, there is no need to even to develop integration for that.
